src/components/dialogs/AdvancedFiltersDialog.js

```diff
@@ -185,66 +214,66 @@ const AdvancedFiltersDialog = ({
               setFilters={setFilters}
             />)
           })}
-          <div 
+          <div
             style={{ backgroundColor: "#DFEDEF", paddingLeft: "10px", paddingBottom: "10px" }}
           >
-            <AddCircle 
-              style={{ 
-                border: "thin solid", 
-                borderRadius: "40px", 
-                width: "16px", 
-                height: "16px" 
-              }} 
+            <AddCircle
+              style={{
+                border: "thin solid",
+                borderRadius: "40px",
+                width: "16px",
+                height: "16px"
+              }}
               onClick={handleAddFilter}
             />
-            <Button 
-              onClick={handleAddFilter} 
+            <Button
+              onClick={handleAddFilter}
               variant="outlined"
-              style={{ 
-                border: "0px", 
-                "marginBottom": "6px", 
-                fontSize: "0.8rem" 
+              style={{
+                border: "0px",
+                "marginBottom": "6px",
+                fontSize: "0.8rem"
               }}
             >
               {formatMessage(intl, "core", "core.advancedFilters.button.addFilters")}
             </Button>
           </div>
         </DialogContent>
-        <DialogActions 
-          style={{ 
-            display: "inline", 
+        <DialogActions
+          style={{
+            display: "inline",
             paddingLeft: "10px",
             marginTop: "25px",
-            marginBottom: "15px"  
+            marginBottom: "15px"
           }}
         >
           <div>
             <div style={{ float: "left" }}>
-              <Button 
-                onClick={handleRemoveFilter} 
+              <Button
+                onClick={handleRemoveFilter}
                 variant="outlined"
                 style={{ border: "0px" }}
               >
                 {formatMessage(intl, "core", "core.advancedFilters.button.clearAllFilters")}
               </Button>
             </div>
-            <div style={{ 
-                float: "right", 
-                paddingRight: "16px" 
+            <div style={{
+                float: "right",
+                paddingRight: "16px"
               }}
             >
-              <Button 
-                onClick={handleClose} 
-                variant="outlined" 
+              <Button
+                onClick={handleClose}
+                variant="outlined"
                 autoFocus
-                style={{ margin: "0 16px" }} 
+                style={{ margin: "0 16px" }}
```

Comment:
As the improvement if we have time now - we can move all css outside of component for better management of styles. But it is optional at this stage.

Reply:
I mean everywhere we have such definitions of css